index.html

```diff
@@ -40,7 +40,7 @@ <h4 class="modal-title">Credits</h4>
 			</div>
 		</div> <!-- END MODAL -->
 
-		<div class="container">
+		<div class="myGrid">
```

Comment:
There are some of styles naming convertions  for css just to let you know https://css-tricks.com/css-style-guides/